lms/djangoapps/dashboard/git_import.py

```diff
@@ -297,8 +298,14 @@ def add_repo(repo, rdir_in, branch=None):
         course_id = match.group(1)
         try:
             course_key = CourseKey.from_string(course_id)
+            course_key = CourseLocator(
```

Comment:
I am converting course key to format "course-v1:org+course+run" instead of previously it was creating a splashed course id, it fixed the problem for new imported courses

Reply:
splashed? Do you mean slash-separated?